### PR TITLE
qa_openstack: Prepare for Kilo tests

### PR DESCRIPTION
**Pull request by toabctl**
Add option to be able to run Kilo packaged version.

### CODE REVIEW

**Comment by jdsn** [comment]
In general ok, but there seems to be a lot code duplication here. Do you think its worth cleaning that up as well?

**Comment by toabctl** [comment]
@jdsn: I would like to postpone the cleanup until Kilo packages are tested

File: scripts/jenkins/qa_openstack.sh [hunk 1]
```diff
@@ -102,6 +102,12 @@ case "$cloudsource" in
             $zypper ar -G -f http://download.opensuse.org/repositories/Cloud:/OpenStack:/Juno:/Staging/$REPO/ cloudhead
         fi
     ;;
+    openstackkilo)
+        $zypper ar -G -f http://download.opensuse.org/repositories/Cloud:/OpenStack:/Kilo/$REPO/ cloud
+        if test -n "$OSHEAD" ; then
+            $zypper ar -G -f http://download.opensuse.org/repositories/Cloud:/OpenStack:/Kilo:/Staging/$REPO/ cloudhead
```
Comment (jdsn) [inline scripts/jenkins/qa_openstack.sh]: Kilo:Staging is empty and so this URL is  not existing yet.

**Comment by jdsn** [comment]
:+1:

**Comment by bmwiedemann** [comment]
:+1:

**Pull request merged**